content/en/docs/concepts/glossary.md

```diff
@@ -118,9 +117,8 @@ pull-based.
 
 ### **Field**
 
-Name/value pairs added to [`Log Records`](#log-record) (similar to
-[`Attributes`](#attribute) for [`Spans`](#span) and [`Labels`](#label) for
-[`Metrics`](#metric)). See [field spec][field].
+Term used specifically for [`Log Records`](#log-record). See
+[`Attribute`](#attribute) and the [field spec][field].
```

Comment:
Attribute then sends folks directly to Metadata. Perhaps we should direct them there right away?

Reply:
I went back and forth here - attribute appears to the word OTel wants to use to mean metadata, but it's not used everywhere. So option is to link everything to attribute, which just links to metadata or direct to metadata. I'm going switching direct to metadata and will update this.